internal/pkg/flink/internal/store/store.go

```diff
@@ -104,34 +104,24 @@ func (s *Store) FetchStatementResults(statement types.ProcessedStatement) (*type
 	}
 
 	// Process remote statements that are now running or completed
-	pageToken := statement.PageToken
-	runningNoTokenRetries := 5
-	for i := 0; i < runningNoTokenRetries; i++ {
-		// TODO: we need to retry a few times on transient errors
-		statementResultObj, err := s.client.GetStatementResults(s.appOptions.GetOrgResourceId(), s.appOptions.GetEnvId(), statement.StatementName, pageToken)
-		if err != nil {
-			return nil, &types.StatementError{Msg: err.Error()}
-		}
+	statementResultObj, err := s.client.GetStatementResults(s.appOptions.GetOrgResourceId(), s.appOptions.GetEnvId(), statement.StatementName, statement.PageToken)
```

Comment:
Nit: Remove ambiguity
```suggestion
	statementResultObj, err := s.client.GetStatementResults(s.appOptions.GetOrgResourceId(), s.appOptions.GetEnvironmentId(), statement.StatementName, statement.PageToken)
```

Reply:
Done!